Assignment/starter_code/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,69 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('h1').on('click', function(){
+	$('h1').slideUp();
+});
 // 2.	Remove the purplebox class
+$('#box14').removeClass('purplebox');
+
+// Do you need to target the specfic box, why does 'body' not work here?
 
 // 3.	Hide all of the div’s in #row4
 
+$('#row4 div').hide();
+
 // 4.	Add a paragraph that says “I have been added” to all boxes with class “add-para” (Hint: you'll need to look up the append() method)
 
+$('.add-para').append('I have been added');
+
 // 5.	Add a class in your css and add it to all items with the class .box
 
+$('.box').addClass('fun')
+
 // 6.	Show the footer when the anchor in #box2 is clicked
 
+$('#box2 a').on('click', function(){
+	$('footer').show();
+});
+
 // 7.	Change the styling of the span inside of #box3 by adding a class (and style that class in CSS)
 
+$('#box3 span').addClass('crazy');
+
 // 8.	Change the background color of all the articles
 
+$('article').css('background-color', 'red');
+
 // 9.	Change the font color of all the articles in row 2
 
+$('#row2 article').css('color', 'white');
+
 // 10.	Select the h4 in #box11 and hide the span inside of #box11 using the .siblings() function
 
+$('#box11 h4').siblings('span').hide();
+
 // 11.	Clicking on #box7 should hide the span within the h2
 
+$('#box7').on('click', function(){
+	$('h2 span').hide();
+});
+
 // 12.	Add a h3 to the beginning of #box12
 
+$('#box12 p').before('<h3>Happy h3 heading</h3>');
```

Comment:
Excellent job! You could also have used `.prepend()` on just the `$('#box12')`, but your method is just as good :+1: